cogs/moveusers.py

```diff
@@ -80,6 +84,8 @@ async def automove(self, ctx):
                                 await self._move_users_to_channel_by_id(members_to_move, DIFFERENT_NAME_CHANNELS["Lounge"])
                             else:
                                 await self._move_users_to_channel_by_id(members_to_move, DIFFERENT_NAME_CHANNELS["Study"])
+                        elif voice_channel_groups[channel]["Streaming"][member_screen_shares] != "General":
```

Comment:
Not the best to do a not operator here. If possible just always stick with positives for easier understanding. Especially since it's a continue, let's do a continue for else and check for general and if so move to lounge or cinema